apps/backend/src/utils/getTransporter.js

```diff
@@ -0,0 +1,37 @@
+const nodemailer = require('nodemailer');
+
+const { SMTP_HOST, SMTP_PORT, SMTP_USER, SMTP_PASSWORD, SMTP_USER_NOTIFY, SMTP_PASSWORD_NOTIFY } =
```

Comment:
@Chifez Please make sure to also update the .env.example files to make sure it is updated with what is expected.